pyrtl/helperfuncs.py

```diff
@@ -795,11 +795,20 @@ def _convert_verilog_str(val: str, bitwidth: int = None,
     if val.startswith('-'):
         neg = True
         val = val[1:]
+
     split_string = val.lower().split("'")
     if len(split_string) != 2:
         raise PyrtlError('error, string not in verilog style format')
     try:
         bitwidth = int(split_string[0])
+        if passed_bitwidth is not None:
+            if bitwidth > passed_bitwidth:
+                raise PyrtlError(
+                    "bitwidth parameter passed (%d) cannot fit Verilog-style constant with bitwidth %d"
```

Comment:
I think we should keep the `(if bitwidth=None is used, PyRTL will determine the bitwidth from the Verilog-style constant specification)` hint in this error message